### PR TITLE
Bug: fix aztk cluster submit paths, imports

### LINKED ISSUE #463: cluster submit broken since storage refactor
command_builder no longer copied to $DOCKER_WORKING_DIR, which breaks the cluster_submit command.

submit.py also moved to a new location.

### PR DESCRIPTION
**Pull request by jafreck**
Fix #463

### CODE REVIEW

File: aztk/node_scripts/docker_main.sh [hunk 1]
```diff
@@ -28,7 +28,8 @@ done
 echo "Starting setup using Docker"
 
 $(pyenv root)/versions/$AZTK_PYTHON_VERSION/bin/pip install -r $(dirname $0)/requirements.txt
-export PYTHONPATH=$PYTHONPATH:$DOCKER_WORKING_DIR
+
+echo 'export PYTHONPATH=$PYTHONPATH:$DOCKER_WORKING_DIR' >> ~/.bashrc
```
Comment (timotheeguerin) [inline aztk/node_scripts/docker_main.sh]: don't you need to source bashrc now? or write the same line above

Comment (jafreck) [reply aztk/node_scripts/docker_main.sh]: yeah sourcing bash.rc is done in submit.py now: [here](https://github.com/Azure/aztk/pull/464/files#diff-05f414f140be1c8a088b89ddb5502a12R84)

Comment (timotheeguerin) [reply aztk/node_scripts/docker_main.sh]: yeah but don't we also need it here for docker_main

Comment (jafreck) [reply aztk/node_scripts/docker_main.sh]: I don't think so, at least not until we do #465 but it won't hurt to do now.

**Pull request merged**